internal/reports/report_helpers.go

```diff
@@ -42,8 +42,8 @@ type tableReport struct {
 }
 
 type tableReportBody struct {
-	Heading string
-	Lines   []string
+	Heading   string
+	IssueBody []Issue
 }
 
```

Comment:
The discrepancy in issue number vs older warning count is coming because we are not taking into account the number of notes & suggestions similar to warnings and error

Reply:
The issue number is getting calculated by iterating the Issues List, the discrepancy is in the warning count.
Resolving this for now, will discuss and commit the required changes further.